src/managers/SessionManager.ts

```diff
@@ -24,6 +27,23 @@ class SessionManager {
         });
         log(MasterTick);
     }
+    async createSession(): Promise<string> {
+        const playerName = PlayerManager.generateName();
+
+        const requestBody = playerName;
+        log(playerName);
+        const response: axios.AxiosResponse<PlayerDTO> = await api.post(
+            "/user",
+            requestBody,
+        );
+        localStorage.setItem("playerId", response.data.playerId);
+        return response.data.sessionId;
+    }
 }
+type PlayerDTO = {
+    username: string;
+    playerId: string;
+    sessionId: string;
+};
```

Comment:
I think it would be a good idea if we put all custom type definitions into a separate file inside the definitions folder.